modules/info.py

```diff
@@ -91,6 +91,13 @@ def get_dot_git(target,response):
         cprint("NO .git folder found", 'blue')
 
 
+def get_bitcoin_address(target, response):
+    bitcoins = re.findall(r'^[13][a-km-zA-HJ-NP-Z1-9]{25,34}$', response)
```

Comment:
What does this line do exactly? Could you add a comment explaining?

Reply:
It checks for bitcoin addresses